recipes/qt/6.x.x/conanfile.py

```diff
@@ -237,7 +237,9 @@ def requirements(self):
             self.requires("pcre2/10.36")
         if self.options.get_safe("with_vulkan"):
             self.requires("vulkan-loader/1.2.172")
-
+            if tools.is_apple_os(self.settings.os):
+                self.requires("moltenvk/1.1.1")
+                self.requires("vulkan-headers/1.2.172", override=True)
```

Comment:
I don't think we should do this. it would be better to upgrade moltenvk's dependencies